pkgs/tools/misc/usbmuxd/default.nix

```diff
@@ -1,23 +1,37 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, libusb1, libimobiledevice }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, libimobiledevice
+, libusb1
+}:
 
 stdenv.mkDerivation rec {
   pname = "usbmuxd";
-  version = "unstable-2021-05-08";
+  version = "1.1.1+date=2022-04-04";
```

Comment:
Please don't invent new version formats.

Reply:
This is an https://github.com/NixOS/rfcs/pull/107 version. I know that RFC hasn't been accepted yet, but it solves the problem of the traditional `unstable` version causing downstream `pkg-config` version checks to fail. I previously had code to manually add the last stable version but as @jtojnar pointed out that's prone to problems when people forget to update it.